sksparse/cholmod.pyx

```diff
@@ -138,6 +140,37 @@ cdef class Factor
 class CholmodError(Exception):
     pass
 
+class CholmodNotPositiveDefiniteError(CholmodError):
+    def __init__(self, message, column=None, factor=None):
+        super().__init__(message)
+        self.column = column
+        self.factor = factor
+
+class CholmodNotInstalledError(CholmodError):
+    def __init__(self, message=""):
+        message = "method not installed: {}".format(message)
+        super().__init__(message)
+
+class CholmoOutOfMemoryError(CholmodError):
```

Comment:
Typo is fixed.
Its not so easy to produce (some of) these errors.
It might be that the error message in `cholmod` already contain the reason for the error. But it is nicer to have for each reason a different `Exception` in Python, so that the code might react depending on the reason and not have to pass the error message somehow which might also change in a future version.